src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,60 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+import { useState } from 'react';
+
+export const Tabs = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const [selectedTab, setSelectedTab]
+  = useState(tabs.find(tab => tab.id === selectedTabId) ?? tabs[0]);
```

Comment:
Why do you need this state?

You have `onTabSelected` prop that change `selectedTabId` state in `App` component, when we change state we re-render our `Tabs` componnet with new `selectedTabId` value, so you can just create a variable like 
```jsx
const selectedTab = tabs.find(tab => tab.id === selectedTabId) ?? tabs[0]
```